README.md

```diff
@@ -68,6 +70,37 @@ Once you have imported fozzie into your Sass, you'll have access to the fozzie v
     ]
   ```
 
+### Using the compiled CSS directly
+Fozzie offers a few pre-compiled CSS files for web applications not using SCSS. These are:
+1. `fozzie-reset.css` - our normalize and reset styles
+2. `fozzie-typography.css` - our typographic styles
+3. `fozzie-utilities.css` - trumps and utility classes to use
+
+All of the CSS files come as regular or minified CSS and include sourcemaps.
+
+Example of using them in a project (note: you can also pull these from `dist/css` if you have installed Fozzie in the project):
+  ```html
+    <!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>A JustEat Marketing Site</title>
+        <!-- use fozzie CSS files to provide base styles and utilities classes used by the cookie banner   -->
+        <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-reset.css" />
+        <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-typography.css" />
+        <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-utilities.css" />
+    </head>
+
+    <body>
+        <h1>Hello World!</h1>
+    </body>
+
+    </html>
+  ```
+
```

Comment:
I would probably have a note on locking the version of those files through unpkg, else if a breaking change were to happen and get released to fozzie, unpkg would simply redirect to the latest major version in the next release build of that website.

The version could be locked for one of these files like:

`https://unpkg.com/@justeat/fozzie@10.17.1/dist/css/fozzie-reset.css`